programs/gummyroll_crud/src/lib.rs

```diff
@@ -4,51 +4,149 @@ use gummyroll::{program::Gummyroll, Node};
 declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 
 #[derive(Accounts)]
-pub struct Add<'info> {
+#[instruction(max_depth: u32, max_buffer_size: u32)]
+pub struct CreateTree<'info> {
+    #[account(mut)]
+    pub authority: Signer<'info>,
+    #[account(
+        init,
+        seeds = [
+            b"gummyroll-crud-authority-pda",
+            merkle_roll.key().as_ref(),
+            authority.key().as_ref(),
+        ],
+        bump,
+        payer = authority,
+        space = 0
+    )]
```

Comment:
You don't need to initialize this account :)

Reply:
![](http://gph.is/1gRoZyv)